examples/guestbook/README.md

```diff
@@ -1,203 +1,253 @@
+$ kubectl create -f redis-master-controller.yaml
+replicationcontrollers/redis-master
+```
+
+The `replicationcontrollers/redis-master` line is the expected response to this operation.
+You can see the replication controllers for your cluster by running:
 
+```shell
 $ kubectl get rc
 CONTROLLER                             CONTAINER(S)            IMAGE(S)                                 SELECTOR                     REPLICAS
 redis-master                           master                  redis                                    name=redis-master            1
 ```
 
-Once that's up you can list the pods in the cluster, to verify that the master is running:
+Then, you can list the pods in the cluster, to verify that the master is running:
 
 ```shell
 $ kubectl get pods
 ```
 
-You'll see all kubernetes components, most importantly the redis master pod. It will also display the machine that the pod is running on once it gets placed (may take up to thirty seconds):
+You'll see all pods in the cluster, including the redis master pod, and the status of each pod.
+The name of the redis master will look similar to that in the following list:
 
 ```shell
-POD                                          IP                  CONTAINER(S)            IMAGE(S)                                 HOST                                                              LABELS                                                     STATUS
-redis-master-controller-gb50a                10.244.3.7          master                  redis                                    kubernetes-minion-7agi.c.hazel-mote-834.internal/104.154.54.203   name=redis-master                                          Running
+NAME                                           READY     REASON    RESTARTS   AGE
+...
+redis-master-dz33o                             1/1       Running   0          2h
 ```
+(Note that an initial `docker pull` to grab a container image may take a few minutes, depending on network conditions. A pod will be reported as `Pending` while its image is being downloaded.)
+
+#### Optional Interlude
 
-If you ssh to that machine, you can run `docker ps` to see the actual pod:
+You can get information about a pod, including the machine that it is running on, via `kubectl describe pods/<pod_name>`.  E.g., for the redis master, you should see something like the following (your pod name will be different):
 
 ```shell
-me@workstation$ gcloud compute ssh kubernetes-minion-7agi
+$ kubectl describe pods/redis-master-dz33o
+...
+Name:       redis-master-dz33o
+Image(s):     redis
+Node:       kubernetes-minion-krxw/10.240.67.201
+Labels:       name=redis-master
+Status:       Running
+Replication Controllers:  redis-master (1/1 replicas created)
+Containers:
+  master:
+    Image:    redis
+    State:    Running
+      Started:    Fri, 12 Jun 2015 12:53:46 -0700
+    Ready:    True
+    Restart Count:  0
+Conditions:
+  Type    Status
+  Ready   True
+No events.
+```
+
```

Comment:
We can introduce "kubectl logs <Pod_name>" here. Basically it's the same as "docker logs <container_id>" but more convenient, as you don't need to ssh.